python_ta/contracts/__init__.py

```diff
@@ -114,13 +130,25 @@ def wrapper(wrapped, instance, args, kwargs):
         try:
             r = _check_function_contracts(wrapped, instance, args, kwargs)
             _check_invariants(instance, rep_invariants, init.__globals__)
+            _check_class_type_annotations(instance)
         except AssertionError as e:
             raise AssertionError(str(e)) from None
         else:
             return r
 
     return wrapper(wrapped)
 
+def _check_class_type_annotations(instance):
```

Comment:
You'll notice I put in type annotations everywhere. You can use `Any` for `instance` and `None` for the return type.